src/utils/formValidationUtils.js

```diff
@@ -0,0 +1,19 @@
+import { isFinite, isNumber } from "lodash"
+
+export const validateNumberWithinRange = ({
+  min = Infinity,
+  max = Infinity,
```

Comment:
Your default for `min` should probably be `-Infinity`, but don't need to be providing defaults